examples/knative/go/kn-go-echo/README.md

```diff
@@ -0,0 +1,121 @@
+# kn-go-echo
+
+Simple Go function running in Knative to echo [CloudEvents](https://github.com/cloudevents/sdk-go).
+In addition, [ko](https://github.com/google/ko) is used to create the artifacts.
+
+Valid events will be printed out JSON-encoded, separated by their CloudEvent
+`attributes` and `data` for better readability.
+
+## Deployment (Knative)
+
+**Note:** The following steps assume a working Knative environment using the
+`default` broker. The Knative `service` and `trigger` will be installed in the
+`default` Kubernetes namespace, assuming that the broker is also available there.
+
+```bash
+# Create the service.
+kn service create kn-go-echo --port 8080 --image vmware/veba-kn-go-echo:latest
+
+# Create the trigger.
+kn trigger create kn-go-echo --broker default --sink ksvc:kn-go-echo
+```
+
+## Build with `ko`
+
+`ko` was created to help Knative developers build images and binaries without
+the need for Dockerfiles nor Makefiles. See more information in [this Knative
+blog](https://knative.dev/blog/2018/12/18/ko-fast-kubernetes-microservice-development-in-go/).
+
+Follow setup instructions at https://github.com/google/ko).
+
+After installing ko and setting the appropriate environmental variables for it,
```

Comment:
```suggestion
After installing ko and setting the appropriate environment variables for it,
```

Reply:
Can you show which variables? Assuming you're talking about `KO_DOCKER_REPO`?

Reply:
Yup, I was talking about that. I'll include it.